ci.nix

```diff
@@ -91,7 +86,7 @@ let
           # FIXME: this should simply be set on the main shell derivation, but this breaks
           # lorri: https://github.com/target/lorri/issues/489. In the mean time, we set it
           # only on the CI version, so that we still catch it, but lorri doesn't see it.
-          shell = (import ./shell.nix { inherit packages; }).overrideAttrs (attrs: attrs // {
+          shell = (import ./dev-shell.nix { inherit packages system; }).overrideAttrs (attrs: attrs // {
             disallowedRequisites = [ marlowe.haskell.packages.plutus-core.components.library ];
           });
```

Comment:
With Flakes, we can likely drop Lorri support, as Lorri doesn't support Flakes anyway (And Flakes have fast cached shells and a direnv integration of their own).